webrender/src/renderer.rs

```diff
@@ -1478,6 +1513,48 @@ impl Renderer {
                                         max_prim_items,
                                         &projection);
                 }
+                &PrimitiveBatchData::ExternalImage(ref ubo_data, img_key) => {
+
+                    let (shader, max_prim_items) = match self.external_textures.get(&img_key) {
+                        Some(&ExternalTexture::Yuv { y_id, u_id, v_id }) => {
+                            self.device.bind_texture(TextureSampler::YPlane, y_id);
+                            self.device.bind_texture(TextureSampler::UPlane, u_id);
+                            self.device.bind_texture(TextureSampler::VPlane, v_id);
+
+                            if has_complex_clip {
+                                // TODO(nical)
+                                unimplemented!();
+                            }
+
+                            // TODO(nical) needs its own GPU_TAG?
+                            self.gpu_profile.add_marker(GPU_TAG_PRIM_IMAGE);
```

Comment:
Yup, let's add a new GPU tag - just so we can see what the shader costs of the YUV conversion are like.